agile-development/intro-to-agile.md

```diff
@@ -54,14 +54,14 @@ The benefits of Agile software development includes:
 - Minimum Viable Product (MVP)
 
 ### Materials
-- [Manifesto for Agile Software Development](http://agilemanifesto.org/)
-- [Scrum](https://www.youtube.com/watch?v=WxiuE-1ujCM)
-- [eXtreme Programming (XP)](https://en.wikipedia.org/wiki/Extreme_programming)
-- [Feature Driven Delopment (FDD)](https://en.wikipedia.org/wiki/Feature-driven_development)
-- [User stories](https://www.mountaingoatsoftware.com/agile/user-stories)
-- [Time-boxing](https://www.agilealliance.org/glossary/iteration)
-- [Kanban board](https://leankit.com/learn/kanban/kanban-board/)
-- [Retrospectives](http://finding-marbles.com/retr-o-mat/what-is-a-retrospective/)
+- [Techtonia sildes: Manifestos for Agile Software Development](https://docs.google.com/presentation/d/1SdCigo6K4aW8p-SZRA0vdI6w628pwhjzf-oX8UKIhS8/edit#slide=id.g1fcf0059a0_0_0)
+- [Techtonica Slides: Scrum vs Kanban in 5 min](https://docs.google.com/presentation/d/1SdCigo6K4aW8p-SZRA0vdI6w628pwhjzf-oX8UKIhS8/edit#slide=id.g1fcf0059a0_0_90)
+     - [Video: Explaining Scrum in less than 120 seconds](https://www.youtube.com/watch?v=WxiuE-1ujCM)
+- [Techtonica Slides: Key Agile Concepts: User stories](https://docs.google.com/presentation/d/1SdCigo6K4aW8p-SZRA0vdI6w628pwhjzf-oX8UKIhS8/edit#slide=id.g1fcf0059a0_0_55)
```

Comment:
Why are the slides being cited multiple times? It can just be once even though it appears throughout the outline

Reply:
I added the links so that they can directly jump on that topic in slide.
But I guess it is not required and will fix it